### ISSUE TITLE
NOAUTH after full queue and reconnect

### ISSUE THREAD

**Issue by mp911de** [lettuce-io/lettuce-core#691]
```
com.lambdaworks.redis.protocol.ConnectionWatchdog[] INFO Reconnecting, last destination was 
com.lambdaworks.redis.protocol.CommandHandler[] WARNING [chid=0x17, channel=0xa1ed60d5, ######## -> ] Unexpected exception during request: com.lambdaworks.redis.RedisException: Request queue size exceeded: 100. Commands are not accepted until the queue size drops.
com.lambdaworks.redis.RedisException: Request queue size exceeded: 100. Commands are not accepted until the queue size drops.
```

**Comment by mp911de** [comment]
Happens if using Lettuce 4.3 (Ping before connect/calling `auth(…)`/passing RedisURI with password). Ping before connect auth is fixed in Lettuce 4.4. Calling `auth(…)` is still broken.

**Comment by mp911de** [comment]
Increasing the number of allowed commands by the maintenance command limit seems to be a good fit for an intermediate fix. More of a patch, see also #697.
